variants/XMC1400/config/XMC1400_Arduino_Kit/pins_arduino.h

```diff
@@ -0,0 +1,290 @@
+/*
+  pins_arduino.h - Pin definition functions for Arduino
+  Part of Arduino - http://www.arduino.cc/
+
+  Copyright (c) 2007 David A. Mellis
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General
+  Public License along with this library; if not, write to the
+  Free Software Foundation, Inc., 59 Temple Place, Suite 330,
+  Boston, MA  02111-1307  USA
+*/
+#ifndef PINS_ARDUINO_H_
+#define PINS_ARDUINO_H_
+
+//****************************************************************************
+// @Project Includes
+//****************************************************************************
+#include <XMC1400.h>
+
+//****************************************************************************
+// @Defines
+//****************************************************************************
+// XMC_BOARD for stringifying into serial or other text outputs/logs
+// Note the actual name XMC and number MUST have a character between
+// to avoid issues with other defined macros e.g. XMC1400
+#define XMC_BOARD           XMC 1400 Kit for Arduino
+
+/* On board LED is ON when digital output is 0, LOW, FALSE, OFF */
+#define  XMC_LED_ON         0
+
+// Following were defines now evaluated by compilation as const variables
+// After definitions of associated mapping arrays
+extern const uint8_t NUM_DIGITAL;
+extern const uint8_t GND;
+extern const uint8_t NUM_PWM4;
+extern const uint8_t NUM_PWM;
+extern const uint8_t NUM_INTERRUPT;
+extern const uint8_t NUM_ANALOG_INPUTS;
+#define NUM_LEDS            3
+#define NUM_SERIAL          1
+#define NUM_TONE_PINS       4
+#define NUM_TASKS_VARIANT   8
+
+// Indicate unit has RTC/Alarm
+#define HAS_RTC             1
+
```

Comment:
Need to change to board agnostic method of gpio reset pin
#define HAS_GPIO_RESET
#define GPIO_RESET_PIN

See conversation on pull request

Reply:
Currently, only two boards XMC 1100 Boot Kit and XMC 1400 Kit for Arduino have dedicated reset pins. Such a boad agnostic approach may be added when more boards with dedicated resets are added to the core.